src/docs/community/migration.mdx

```diff
@@ -0,0 +1,242 @@
+...
+<div style={{background: Colors.blue['500']}}/>
+```
+
+## @brightlayer-ui/cra-template-authentication
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   You can use these templates with the Brightlayer UI CLI:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new react --template=authentication
+    ```
+
+## @brightlayer-ui/cra-template-authentication-typescript
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   You can use these templates with the Brightlayer UI CLI:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new react --template=authentication
+    ```
+
+## @brightlayer-ui/cra-template-blank
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/cra-template-blank-typescript
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/cra-template-routing
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/cra-template-routing-typescript
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/eslint-config
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/icons
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/icons-mui
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/icons-svg
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/mapbox
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/ng-progress-icons
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/prettier-config
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/publish
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/react-auth-shared
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/react-auth-workflow
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-components
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   For using App Bar in app, you can use `import { AppBar } from '@brightlayer-ui/react-components';`
+
+## @brightlayer-ui/react-native-progress-icons
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-native-template-authentication
+
+-   `pxb` preface changed to `blui`.
+-   You can use these templates with the Brightlayer UI CLI:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new react-native --cli=rnc --template=authentication
+    ```
+
+## @brightlayer-ui/react-native-template-authentication-typescript
+
+-   `pxb` preface changed to `blui`.
```

Comment:
What prefix?

Reply:
I think you get the idea about this, so I won't comment on all of these. But I don't know what you mean by the pxb preface on a lot of these (and I authored a lot of them). The only package I know of where we explicitly prefix anything is in the angular components. Before/after examples will go a long way to communicate what this means.